alpha/apps/kaltura/modules/extwidget/actions/serveFlavorAction.class.php

```diff
@@ -280,27 +289,32 @@ protected function serveEntriesAsPlaylist($entryIds, $durations, $referenceEntry
 		// build the media set
 		if ($isLive)
 		{
-			$mediaSet = $this->serveLivePlaylist($durations, $sequences,
-				$playlistStartTime, $firstClipStartTime, $initialClipIndex, $initialSegmentIndex, $origEntry->getRepeat());
+			$repeat = !!$origEntry->getRepeat(); // Convert to bool
```

Comment:
What was intention here ? convert to boolean? the line is not readable , better use unary term -  $origEntry->getRepeat() ? true : false